Runtime/Core/ShapeGenerator.cs

```diff
@@ -1,71 +1,8 @@
-using UnityEngine;
-using System.Collections.Generic;
 using UnityEngine.ProBuilder.MeshOperations;
+using System;
 
 namespace UnityEngine.ProBuilder
 {
-    /// <summary>
```

Comment:
because this is public API, it needs to go through a deprecation release before we can remove it